sirepo/package_data/static/json/impactt-schema.json

```diff
@@ -0,0 +1,206 @@
+        ]
+    },
+    "localRoutes": {
+        "source": {
+            "config": {
+                "controller": "impacttSourceController as source",
+                "templateUrl": "/static/html/ipmactt-source.html"
+            }
+        }
+    },
+    "model": {
+        "simulationStatus": {},
+        "impacttReport": {},
+        "beam": {
+            "n_p_row": ["Nprow", "Integer", 2, "Number of columns of processors, used to decompose domain along Z dimension"],
+            "n_p_col": ["Npcol", "Integer", 2, "Number of columns of processors, used to decompose domain along Y dimension"],
+            "d_t": ["Dt", "Float", 0, "Time step size (secs)."],
+            "n_step": ["Ntstep ", "Integer", 100000, "Maximum number of time steps."],
+            "n_bunch": ["Nbunch", "Integer", 1],
+            "dim": ["Dim", "Integer", 1, "Random Seed Integer > 0"],
+            "n_p": ["Np", "Integer", 1000, "Number of macroparticles to track"],
+            "flag_map": ["Flagmap", "Integer", 1, "Type of integrator. Currently must be set to 1."],
+            "flag_err": ["Flagerr", "Integer", 1, "Error study flag  0 - no misalignment and rotation errors; 1 - misalignment and rotation errors are allowed for Quadrupole, Multipole (Sextupole, Octupole, Decapole) and SolRF elements."],
+            "flag_diag": ["Flagdiag", "Integer", 1, "Diagnostics flag: 1 - output the information at given time, 2 - output the information at the location of bunch centroid by drifting the particles to that location, 3 or more - no output."],
+            "flag_img": ["Flagimg", "Integer", 0, "Image charge flag. If set to 1 then the image charge forces due to the cathode are included."],
+            "z_img": ["Zimage", "Float", 0.02, "z position beyond which image charge forces are neglected."],
+            "n_x": ["Nx", "Integer", 32, "Number of mesh points in x"],
+            "n_y": ["Ny", "Integer", 32, "Number of mesh points in y"],
+            "n_z": ["Nz", "Integer", 32, "Number of mesh points in z"],
+            "flag_bc": ["Flagbc", "Integer", 1, "Field boundary condition flag: Currently must be set to 1 which corresponds to an open boundary condition."],
+            "x_rad": ["Xrad", "Float", 0.015, "Size of computational domain"],
+            "y_rad": ["Yrad", "Float", 0.015, "Size of computational domain"],
+            "perd_len": ["Perdlen", "Float", 45.0, "Size of computational domain"],
+            "flag_dist": ["FlagDist", "DistType", "Gaussian", "Type of the initial distribution."],
+            "r_start_flg": ["Rstartflag", "Integer", 0, "If restartflag = 1, restart the simulation from the previous check point. If restartflag = 0, start the simulation from the beginning."],
+            "n_emission": ["Nemission", "Integer", 400, "The number of numerical emission steps"],
+            "t_emission": ["Temission [sec]", "Float", 6.5e-12, "Temission Laser pulse emission time"],
+            "distribution_parameters": {
```

Comment:
Move the "distribution_parameters" to the outer level, and you could call it "command_distribution". Nested model definitions aren't supported.